spec/models/ability_spec.rb

```diff
@@ -71,6 +71,22 @@
     end
   end
 
+  %i[tutor organizer].each do |role|
+    it "can view and add notes to application letters" do
```

Comment:
I see what you mean. I didn't expect the {role} to evaluate to the actual role, so I didn't get your point. I will change that.